web/src/LogStore.ts

```diff
@@ -266,6 +280,15 @@ class LogStore {
     if (isStartingNewLine) {
       span.lastLineIndex = this.lines.length
       this.lines.push(candidate)
+
+      // If this starts a warning or error, index it now.
+      let level = newSegment.level
+      if (
+        newSegment.anchor &&
+        (level === LogLevel.WARN || level === LogLevel.ERROR)
+      ) {
+        span.alerts.push({ level: level, lineIndex: this.lines.length })
```

Comment:
AFAICT we never use the `lineIndex` so it's not causing problems, but I think it should be `span.lastLineIndex` because the lines have already been pushed to `this.lines`, so this will be the index _after_ the alert, no?

Reply:
oops, yes!

you're right it's not used now; it was just much easier to debug when this object had some info about where the alert came from